subsys/bluetooth/host/att.c

```diff
@@ -158,8 +157,6 @@ void att_pdu_sent(struct bt_conn *conn, void *user_data)
 		if (!att_send(conn, buf, NULL, NULL)) {
 			return;
 		}
-		/* If the buffer cannot be send unref it */
-		net_buf_unref(buf);
```

Comment:
With this change we now need to handle the failure of `bt_smp_sign` inside of `att_send` to release the buffer.

Reply:
Just fixed that.